WooCommerce/Classes/ServiceLocator/ProductImageUploader.swift

```diff
@@ -179,9 +188,18 @@ private extension ProductImageUploader {
             guard let self = self else { return }
 
             if let error = error, self.statusUpdatesExcludedProductKeys.contains(key) == false {
-                self.errorsSubject.send(.init(siteID: key.siteID, productID: key.productID, productImageStatuses: productImageStatuses, error: error))
+                self.errorsSubject.send(.init(siteID: key.siteID,
+                                              productID: key.productID,
+                                              productImageStatuses: productImageStatuses,
+                                              error: .actionHandler(error: error)))
             }
         }
         statusUpdatesSubscriptions.insert(observationToken)
     }
 }
+
+/// Possible errors from background image upload.
+enum ProductImageUploaderError: Error {
+    case savingProductImages(error: Error)
+    case actionHandler(error: Error)
```

Comment:
Could we use alternate names for these errors?
Something like the following,
```swift
enum ProductImageUploaderError: Error {
    case failedSavingProductAfterImageUpload(error: Error)
    case failedUploadingImage(error: Error)
}
```

Reply:
Sure, I updated the case names as you suggested in https://github.com/woocommerce/woocommerce-ios/pull/7154/commits/eeab67f1d097fa04e1f895c2a6d50bb09e014002